SPECS/python-werkzeug/python-werkzeug.spec

```diff
@@ -1,13 +1,18 @@
 Summary:        The Swiss Army knife of Python web development
 Name:           python-werkzeug
-Version:        2.2.3
+Version:        2.3.7
 Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Languages/Python
 URL:            https://github.com/pallets/werkzeug
 Source0:        https://github.com/pallets/werkzeug/archive/%{version}.tar.gz#/werkzeug-%{version}.tar.gz
+Patch0:         0001-enable-tests-in-rpm-env.patch
+# Necessary because RPM build environment creates specific scenario that violates a core 
+# assumption the testing infrastructure of werkzeug makes
```

Comment:
I'm fine with a surgical disablement of the tests. I'd only ask to expand the comment a bit, to explain what the "specific scenario" and the "core assumption" are. This way, if we make any changes to the build environment in the future, it's going to be easier to tell, if we can try enable these tests.

Reply:
Done